nncf/experimental/post_training/quantization/algorithm.py

```diff
@@ -0,0 +1,168 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import List
+from typing import Dict
+from typing import Optional
+
+from nncf.common.utils.ordered_enum import OrderedEnum
+from nncf.common.quantization.structs import QuantizerConfig
+from nncf.common.quantization.structs import QuantizationMode
+from nncf.common.graph.transformations.layout import TransformationLayout
+
+from nncf.experimental.post_training.backend import BACKEND
+from nncf.experimental.post_training.api.engine import Engine
+from nncf.experimental.post_training.graph.model_transformer import ModelTransformer
+from nncf.experimental.post_training.algorithm import PostTrainingAlgorithm
+from nncf.experimental.post_training.algorithm import PostTraniningAlgorithmParameters
+from nncf.experimental.post_training.compressed_model import CompressedModel
+
+from nncf.experimental.post_training.initialization.algorithm import InitializationAlgorithm
+from nncf.experimental.post_training.initialization.algorithm import InitializationAlgorithms
+from nncf.experimental.post_training.initialization.algorithm import InitizalizationParameters
+from nncf.experimental.post_training.initialization.quantizer_range_finder import QuantizerRangeFinderParameters
+
+from nncf.experimental.post_training.utils import merge_two_dicts
+
+
+class PostTrainingQuantizationParameters(PostTraniningAlgorithmParameters):
+    def __init__(self,
+                 preset: str = 'perfomance',
+                 iterations_number: int = 300,
```

Comment:
I think it is not about iterations but more about the number of samples used in the method. This also should correspond to the number of samples in the DataLoader (should be less or equal). `stat_subset_size` from POT is not the best name but we can call it like `num_samples`.